src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-//
+.PostInfo {
```

Comment:
You should import this style file inside your component. 
There are no styles for `PostInfo` now